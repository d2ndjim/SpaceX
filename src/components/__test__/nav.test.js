import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Nav from '../nav';

describe('NavBar test', () => {
  it('renders navbar', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Nav />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
