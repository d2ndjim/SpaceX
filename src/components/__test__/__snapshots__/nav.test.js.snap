// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar test renders navbar 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="text"
    >
      <img
        alt="logo"
        className="img"
        src="planet.png"
      />
      <h2
        className="h2"
      >
        Space Traveler's Hub
      </h2>
    </div>
    <nav
      className="nav"
    >
      <a
        aria-current="page"
        className="active active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
      <a
        className="links"
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
      <a
        className="links"
        href="/dragons"
        onClick={[Function]}
      >
        Dragons
      </a>
      <a
        className="links"
        href="/profile"
        onClick={[Function]}
      >
        My profile
      </a>
    </nav>
  </header>,
  <hr
    className="line"
  />,
]
`;
